app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,74 @@
+class TweeetsController < ApplicationController
+  get '/tweets/new' do
+    if !session[:user_id]
```

Comment:
I would set up a helper method in the application controller to check if a user is logged_in?

examples of this would be 

```ruby
helpers do
    def logged_in?
      !!session[:user_id]
    end

    def current_user
      User.find(session[:user_id])
    end

  end
```